src/libraries/System.Text.Json/tests/System.Text.Json.FSharp.Tests/EnumTests.fs

```diff
@@ -15,6 +15,13 @@ type BadEnum =
 let badEnum = BadEnum.``There's a comma, in my name`` ||| BadEnum.``There's a comma, even here``
 let badEnumJsonStr = $"\"{badEnum}\""
 
+type NumericLabelEnum =
+  | ``1`` = 1
+  | ``2`` = 2
+  | ``3`` = 4
+  
+let enumWithNonMatchedUnderlyingNumericValueJsonStr = "\"3\""
```

Comment:
Can be inlined in the one test that uses it?

Reply:
Fixed.